pkg/controller/controller_utils_test.go

```diff
@@ -38,6 +38,12 @@ import (
 	"k8s.io/kubernetes/pkg/util/sets"
 )
 
+var (
+	// check structs conform to PodControlInterface
```

Comment:
the idiom is to have these assertions at the declaration site for the type, partially as a hint to hyman readers about what interfaces a type is supposed to implement. Relocate?